beamer/tests/agent/unit/test_handlers.py

```diff
@@ -25,8 +23,10 @@
 )
 from beamer.tests.agent.utils import make_address
 from beamer.tests.constants import FILL_ID
+from beamer.typing import FillId, Termination
+from beamer.util import get_ERC20_abi
 
-ERC2O_ABI = load_ERC20_abi()
+ERC2O_ABI = get_ERC20_abi()
```

Comment:
Now that we load the ERC20 abi just once, we can remove this global and instead just call `get_ERC20_abi`.